play-2.4/swagger-play2/app/play/modules/swagger/SwaggerPlugin.scala

```diff
@@ -80,6 +80,24 @@ class SwaggerPluginImpl @Inject()(lifecycle: ApplicationLifecycle, router: Route
     case Some(value)=> value
   }
 
+  // Play seems to throw an exception if this value is not defined and you get it
```

Comment:
Looked at the Gitter chat - I suppose this plugin could define a default "routes" in the reference.conf and then we know it would be defined.